tests/terraform/create_hdd_controller.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: create a qcow2 image with all the needed tools for tests using
+#          Terraform.
+#
+# Maintainer: Jose Lausuch <jalausuch@suse.de>
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    my ($self) = @_;
+
+    $self->select_serial_terminal;
+
+    zypper_call('-q ar -G  https://download.opensuse.org/repositories/systemsmanagement:/sumaform/openSUSE_Leap_15.0/systemsmanagement:sumaform.repo');
```

Comment:
Maybe [generate_version()](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/repo_tools.pm#L191) can help here, to be more flexible.

Reply:
actually not possible. This will return 'openSUSE_LEAP_15.1' and that repo is not created yet 
https://download.opensuse.org/repositories/systemsmanagement:/sumaform/
only, 15.0.

Same happens in SLE15-SP1. Only SLE-15 exist, so we need to hard code :(

Reply:
Eventually, I am forcing to use SLE15 repo

Reply:
Sounds like, it would make sense to have some sane default, but having a variable to quickly change it makes sense.